utilities/write_batch_with_index/write_batch_with_index_internal.h

```diff
@@ -30,7 +31,7 @@ struct WriteBatchIndexEntry {
 
   // If this flag appears in the offset, it indicates a key that is smaller
   // than any other entry for the same column family
-  static const size_t kFlagMin = std::numeric_limits<size_t>::max();
+  static const size_t kFlagMin = port::kMaxUint64;
```

Comment:
Look at it again. Is there way to support 32-bit? Is http://stackoverflow.com/questions/27442885/syntax-error-with-stdnumeric-limitsmax related? Can we do something like that?